provider/fs/Cargo.toml

```diff
@@ -55,11 +55,16 @@ static_assertions = { version = "1.1", optional = true }
 icu_benchmark_macros = { version = "0.3", path = "../../tools/benchmark/macros" }
 icu_locid_macros = { version = "0.3", path = "../../components/locid/macros" }
 icu_plurals = { version = "0.3", path = "../../components/plurals" }
+serde-json-core = { version = "0.4", features = ["std"] }
 criterion = "0.3.3"
 
 [features]
+# Enables JSON or Bincode data storage
+provider_json = ["serde_json"]
+provider_bincode = ["bincode", "std"]
```

Comment:
I had to make provider_bincode instead of just bincode because it needs the std feature enabled as well.

Reply:
grumble: really wish cargo features supported this better